ext/pdo/tests/pdo_stmt_fetchobject.phpt

```diff
@@ -0,0 +1,107 @@
+--TEST--
+PDO Common: PDOStatement->fetchObject()
+--EXTENSIONS--
+pdo
+--SKIPIF--
+<?php
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+require_once $dir . 'pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+if (getenv('REDIR_TEST_DIR') === false) putenv('REDIR_TEST_DIR='.__DIR__ . '/../../pdo/tests/');
+require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
+$db = PDOTest::factory();
+$db->setAttribute(PDO::ATTR_STRINGIFY_FETCHES, true);
+
+$table = 'pdo_stmt_fetchobject';
+$db->exec("CREATE TABLE {$table} (id INT, label CHAR(1), PRIMARY KEY(id))");
+$db->exec("INSERT INTO {$table} (id, label) VALUES (1, 'a')");
+$db->exec("INSERT INTO {$table} (id, label) VALUES (2, 'b')");
+$db->exec("INSERT INTO {$table} (id, label) VALUES (3, 'c')");
+
+$query = "SELECT id, '', NULL FROM {$table}";
+$stmt = $db->prepare($query);
+
+#[AllowDynamicProperties]
```

Comment:
What happens when this is not present?

Reply:
It will cause a deprecation warning, fetching into an object assigns the value of the column to a property named after the column